R/databricks-utils.R

```diff
@@ -42,18 +42,17 @@ databricks_token <- function(token = NULL, fail = FALSE) {
       }
     }
   }
-  # Checks for OAuth Databricks token inside the RStudio API
-  if (is.null(token) && exists(".rs.api.getDatabricksToken")) {
-    getDatabricksToken <- get(".rs.api.getDatabricksToken")
-    token <- set_names(getDatabricksToken(databricks_host()), "oauth")
-  }
+  # # Checks for OAuth Databricks token inside the RStudio API
+  # if (is.null(token) && exists(".rs.api.getDatabricksToken")) {
+  #   getDatabricksToken <- get(".rs.api.getDatabricksToken")
+  #   token <- set_names(getDatabricksToken(databricks_host()), "oauth")
+  # }
```

Comment:
This **_should_** be handled by SDK config component.

Reply:
Hey, are we talking about this SDK? https://github.com/databricks/databricks-sdk-py/ And if so, can you point me to where it handles the RStudio token? I can't seem to find it

Reply:
SDK won't detect the `.rs.api.getDatabricks*` but maybe theres a gap in my understanding, I thought connect would also write to a config file as well, which the SDK should pickup?